Unable to Compile with Jan 20th 2021 Commits

Errors compiled from current master..Error below.. Rolled back to Commit from Jan 19th and compiles fine.

Scanning dependencies of target dbz2
[ 39%] Building C object src/dbz2/CMakeFiles/dbz2.dir/dbz2.c.o
/xfers/mpifileutils/mpifileutils/src/dbz2/dbz2.c: In function ‘main’:
/xfers/mpifileutils/mpifileutils/src/dbz2/dbz2.c:165:9: error: too many arguments to function ‘mfu_param_path_set_all’
  165 |         mfu_param_path_set_all((uint64_t)numpaths, (const char**)p, paths, mfu_file);
      |         ^~~~~~~~~~~~~~~~~~~~~~
In file included from /xfers/mpifileutils/install/include/mfu_io.h:42,
                 from /xfers/mpifileutils/install/include/mfu_util.h:36,
                 from /xfers/mpifileutils/install/include/mfu.h:24,
                 from /xfers/mpifileutils/mpifileutils/src/dbz2/dbz2.c:16:
/xfers/mpifileutils/install/include/mfu_param_path.h:78:6: note: declared here
   78 | void mfu_param_path_set_all(uint64_t num, const char** paths, mfu_param_path* params);
      |      ^~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/dbz2/CMakeFiles/dbz2.dir/build.make:63: src/dbz2/CMakeFiles/dbz2.dir/dbz2.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:665: src/dbz2/CMakeFiles/dbz2.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

I blew everything away and started over... the above error is gone... but i still cant compile... 

this is the current error and i have this error on two different systems.. one ubuntu 20.04 and 18.04

/xfers/mpifileutils/mpifileutils/src/dtar/dtar.c: In function ‘main’:
/xfers/mpifileutils/mpifileutils/src/dtar/dtar.c:389:5: error: too few arguments to function ‘mfu_param_path_set’
  389 |     mfu_param_path_set(cwd, &cwd_param);
      |     ^~~~~~~~~~~~~~~~~~
In file included from /xfers/mpifileutils/mpifileutils/src/common/mfu_io.h:42,
                 from /xfers/mpifileutils/mpifileutils/src/common/mfu_util.h:36,
                 from /xfers/mpifileutils/mpifileutils/src/common/mfu.h:24,
                 from /xfers/mpifileutils/mpifileutils/src/dtar/dtar.c:20:
/xfers/mpifileutils/mpifileutils/src/common/mfu_param_path.h:69:6: note: declared here
   69 | void mfu_param_path_set(const char* path, mfu_param_path* param, mfu_file_t* mfu_file);
      |      ^~~~~~~~~~~~~~~~~~
make[2]: *** [src/dtar/CMakeFiles/dtar.dir/build.make:63: src/dtar/CMakeFiles/dtar.dir/dtar.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1124: src/dtar/CMakeFiles/dtar.dir/all] Error 2
make: *** [Makefile:130: all] Error 2

--- a/.travis.yml
+++ b/.travis.yml
@@ -47,7 +47,7 @@ before_script:
 
 script:
  - mkdir build install && cd build
- - cmake ../ -DCMAKE_INSTALL_PREFIX=../install
+ - cmake ../ -DCMAKE_INSTALL_PREFIX=../install -DENABLE_EXPERIMENTAL=ON
  - make
  - make test
  - make install

--- a/src/dtar/dtar.c
+++ b/src/dtar/dtar.c
@@ -386,7 +386,7 @@ int main(int argc, char** argv)
     mfu_param_path cwd_param;
     char cwd[PATH_MAX];
     mfu_getcwd(cwd, PATH_MAX);
-    mfu_param_path_set(cwd, &cwd_param);
+    mfu_param_path_set(cwd, &cwd_param, mfu_src_file);
 
     int ret = MFU_SUCCESS;
     if (opts_create) {

